common-client/src/main/scala/org/genivi/sota/core/FakeDeviceRegistry.scala

```diff
@@ -129,4 +129,8 @@ class FakeDeviceRegistry(namespace: Namespace)
   def addGroup(group: Uuid, devices: Seq[Uuid]): Unit = {
     groups.put(group, devices)
   }
+
+  def setInstalledPackages
+  (device: Uuid, json: io.circe.Json)(implicit ec: ExecutionContext) : Future[Unit] =
```

Comment:
Any way to use List instead of json, and still be backward compatible to rvi for now?